identity-service/sequelize/migrations/20210824180845-add-email-index-users-table.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addIndex('Users', ['email'])
```

Comment:
does this by default give the name users_email? would make sense

Reply:
Yup!

![image](https://user-images.githubusercontent.com/19916043/130681079-cab91a70-b8ac-4e87-9b38-5dd558a86645.png)